modules/apps/portal-security/portal-security-auto-login/src/main/java/com/liferay/portal/security/auto/login/remember/me/RememberMeAutoLogin.java

```diff
@@ -106,7 +106,9 @@ protected String[] doLogin(
 
 			long userId = GetterUtil.getLong(credentials[0]);
 
-			if (guestUser.getUserId() == userId) {
+			User realUser = _userLocalService.getUserById(userId);
```

Comment:
We can simply name the variable `user` because there is no impersonation related code here.